quickwit/quickwit-grpc-clients/README.md

```diff
@@ -0,0 +1,6 @@
+# Quickwit gRPC clients
```

Comment:
I will probably rename the crate as `quickwit-grpc-utils` at putting `clients` is misleading.